scripts/kfserving-install.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+if kubectl get svc kfserving-controller-manager-service -n kfserving-system >/dev/null 2>&1 ; then
+  echo "kfserving already installed"
+  exit
+fi
+
+export KFSERVING_VERSION=v0.5.1
+
+curl https://raw.githubusercontent.com/kubeflow/kfserving/${KFSERVING_VERSION}/install/${KFSERVING_VERSION}/kfserving.yaml | \
+  sed 's/cluster-local/knative-local/' | kubectl apply -f -
```

Comment:
what is this for?

Reply:
kfserving expects the knative gateways to be named as `cluster-local`, however with the newer versions of knative its name changed to `knative-local`